recipes/dlib/all/conanfile.py

```diff
@@ -23,7 +23,8 @@ class DlibConan(ConanFile):
         "with_png": [True, False],
         "with_sse2": [True, False, "auto"],
         "with_sse4": [True, False, "auto"],
-        "with_avx": [True, False, "auto"]
+        "with_avx": [True, False, "auto"],
+        "with_openblas": [True, False]
```

Comment:
Add trailing `,`, next time someone adds another function it will show just one line diff

```suggestion
        "with_openblas": [True, False],
```